### PR TITLE
feat: Rename "register" to "mint"

### PR DESCRIPTION
**Pull request by mohandast52**
a

b

c

DOUBT: what should I rename `Registration` to?

d

### CODE REVIEW

**Comment by jmoreira-valory** [comment]
"Pre-Registration, Active Registration, Finished Registration" states in the protocol should NOT change (these registration refer to registration of agent instances in the service).

The idea of changing "Register a Service" etc. is precisely to avoid the confusion between "registering a service in the Autonolas Protocol"   VS   "register agent instances in the service".   The first action is actually "minting an NFT that represents the service", whereas the second action simply associates addresses to an already minted
 service.

File: common-util/List/ListCommon.jsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ DependencyLabel.defaultProps = { type: 'component' };
 
 export const RegisterMessage = ({ handleCancel }) => (
   <RegisterFooter>
-    <p>To register, connect to wallet</p>
+    <p>To mint, connect to wallet</p>
```
Comment (jmoreira-valory) [inline common-util/List/ListCommon.jsx:85]: connect *your* wallet ?

File: common-util/List/RegisterForm/index.jsx [hunk 2]
```diff
@@ -154,7 +154,7 @@ const RegisterForm = ({
           </Form.Item>
         ) : (
           <RegisterFooter>
-            <p>To register, connect to wallet</p>
+            <p>To mint, connect to wallet</p>
```
Comment (jmoreira-valory) [inline common-util/List/RegisterForm/index.jsx:157]: *your

File: components/ListOperators/register.jsx [hunk 3]
```diff
@@ -13,9 +13,9 @@ const RegisterOperators = () => {
 
   return (
     <>
-      <Title level={2}>Register Operator</Title>
+      <Title level={2}>Mint Operator</Title>
```
Comment (jmoreira-valory) [inline components/ListOperators/register.jsx:16]: I am unsure, but if this refers to register an operator on an already minted state, it should be 'register', no?

File: components/ListOperators/register.jsx [hunk 4]
```diff
@@ -13,9 +13,9 @@ const RegisterOperators = () => {
...
       <RegisterFooter>
-        <p>To register, connect to wallet</p>
+        <p>To mint, connect to wallet</p>
```
Comment (jmoreira-valory) [inline components/ListOperators/register.jsx:18]: *your

File: components/ListServices/RegisterForm.jsx [hunk 5]
```diff
@@ -276,7 +276,7 @@ const RegisterForm = ({
           </Form.Item>
         ) : (
           <RegisterFooter>
-            <p>To register, connect to wallet</p>
+            <p>To mint, connect to wallet</p>
```
Comment (jmoreira-valory) [inline components/ListServices/RegisterForm.jsx:279]: *your

**Review by jmoreira-valory — CHANGES_REQUESTED** [review]
See comments

File: common-util/Details/ServiceState/3rdStepFinishedRegistration/index.jsx [hunk 6]
```diff
@@ -16,8 +16,8 @@ import { handleMultisigSubmit } from './utils';
 import { RadioLabel } from '../styles';
 
 const STEP = 3;
-const OPTION_1 = 'Creates a new service multisig with currently registered agent instances';
-const OPTION_2 = 'Updates an existent service multisig with currently registered agent instances. Please note that the only service multisig owner must be the current service owner address';
+const OPTION_1 = 'Creates a new service multisig with currently minted agent instances';
```
Comment (DavidMinarsch) [inline common-util/Details/ServiceState/3rdStepFinishedRegistration/index.jsx:19]:
```suggestion
const OPTION_1 = 'Creates a new service multisig with currently registered agent instances';
```
revert

File: common-util/Details/ServiceState/3rdStepFinishedRegistration/index.jsx [hunk 7]
```diff
@@ -16,8 +16,8 @@ import { handleMultisigSubmit } from './utils';
...
+const OPTION_2 = 'Updates an existent service multisig with currently minted agent instances. Please note that the only service multisig owner must be the current service owner address';
```
Comment (DavidMinarsch) [inline common-util/Details/ServiceState/3rdStepFinishedRegistration/index.jsx:20]:
```suggestion
const OPTION_2 = 'Updates an existent service multisig with currently registered agent instances. Please note that the only service multisig owner must be the current service owner address';
```
revert

File: common-util/Details/ServiceState/utils.jsx [hunk 8]
```diff
@@ -151,7 +151,7 @@ export const onStep2RegisterAgents = ({
         .send({ from: account, value: `${totalBonds}` })
         .then((information) => {
           resolve(information);
-          notifySuccess('Registered Successfully');
+          notifySuccess('Minted Successfully');
```
Comment (DavidMinarsch) [inline common-util/Details/ServiceState/utils.jsx:154]: revert

File: common-util/List/ListCommon.jsx [hunk 9]
```diff
@@ -82,7 +82,7 @@ DependencyLabel.defaultProps = { type: 'component' };
 
 export const RegisterMessage = ({ handleCancel }) => (
```
Comment (DavidMinarsch) [inline common-util/List/ListCommon.jsx:83]: Mint should only be used on original NFT creation, not sure where these messages below come into play. Please check

File: components/ListOperators/index.jsx [hunk 10]
```diff
@@ -27,7 +27,7 @@ const ListOperators = () => {
           type="primary"
           onClick={() => router.push(URL.REGISTER_OPERATOR)}
         >
-          Register
+          Mint
```
Comment (DavidMinarsch) [inline components/ListOperators/index.jsx:30]: Here it should be register

File: components/ListOperators/register.jsx [hunk 11]
```diff
@@ -15,7 +15,7 @@ const RegisterOperators = () => {
     <>
       <Title level={2}>Register Operator</Title>
       <RegisterFooter>
-        <p>To register, connect to wallet</p>
+        <p>To mint, connect your wallet</p>
```
Comment (DavidMinarsch) [inline components/ListOperators/register.jsx:18]: revert to register

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]
Suggest you start this PR fresh. It's a mess to review. @jmoreira-valory please provide more accurate instructions

**Pull request closed without merging**